src/tabs/tab-header.component.ts

```diff
@@ -0,0 +1,103 @@
+import {
+	Component,
+	Input,
+	ViewChild,
+	ElementRef,
+	TemplateRef,
+	Output,
+	OnInit
+} from "@angular/core";
+
+import { Tab } from "./tab.component";
+import { EventEmitter } from "@angular/core";
+
+@Component({
+	selector: "ibm-tab-header",
+	template: `
+		<li
+			[ngClass]="{
+				'bx--tabs__nav-item--selected': active,
+				'bx--tabs__nav-item--disabled': disabled
+			}"
+			class="bx--tabs__nav-item"
+			role="presentation"
+			(click)="selectTab()">
+			<a
+				#tabItem
+				[attr.aria-selected]="active"
+				draggable="false"
+				class="bx--tabs__nav-link"
+				href="javascript:void(0)"
+				[attr.tabindex]="(active? 0 : -1)"
+				role="tab">
+				<ng-container *ngIf="!isTemplate(heading)">
+					{{ heading }}
+				</ng-container>
+				<ng-template
+					*ngIf="isTemplate(heading)"
+					[ngTemplateOutlet]="heading"
+					[ngTemplateOutletContext]="{$implicit: context}">
+				</ng-template>
```

Comment:
Since the header is all net new code, could we use `ng-content` instead of templates? That way we don't have to worry about passing around all sorts of context for the header

Reply:
Yeah that would make the component smaller